opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservableIndicators.js

```diff
@@ -300,11 +295,15 @@ const StixCyberObservableIndicators = createFragmentContainer(
     stixCyberObservable: graphql`
       fragment StixCyberObservableIndicators_stixCyberObservable on StixCyberObservable {
         id
-        indicators(first: 200) @connection(key: "Pagination_indicators") {
+        observable_value
+        parent_types
+        entity_type
+        indicators(first: 200) {
```

Comment:
Looks like is th 200 from here. Without a constant, difficult to think about changing at 2 different places.